test/test_shape.py

```diff
@@ -214,3 +215,60 @@ def _m1bm1_dim_checker(x: TensorType[-1, "b":4, -1]):
         _abm1_dim_checker(x)
     with pytest.raises(TypeError):
         _m1bm1_dim_checker(x)
+
+
+def test_any_dim():
+    @typeguard.typechecked
+    def _3any_dim_checker(x: TensorType[3, Any]):
+        pass
+
+    @typeguard.typechecked
+    def _any4_dim_checker(x: TensorType[Any, 4]):
+        pass
+
+    @typeguard.typechecked
+    def _anyany_dim_checker(x: TensorType[Any, Any]):
+        pass
+
+    @typeguard.typechecked
+    def _34any_dim_checker(x: TensorType[3, 4, Any]):
+        pass
+
+    @typeguard.typechecked
+    def _m14any_dim_checker(x: TensorType[Any, 4, Any]):
```

Comment:
This function should be called `_any4any_dim_checker`.

Reply:
Fixed :smile: